R/repo.R

```diff
@@ -20,9 +20,11 @@ repo_source <- function(pkg, paths) {
     return()
   }
 
+  needs_slash <- !grepl("/$", url$source) & !grepl("^/", paths)
```

Comment:
give a name to `!grepl("^/", paths)` to explauin why it's needed? `not_relative_links`?

Reply:
also should it be `need_slashes` or `need_slash` given it's applied to several things at once :nail_care:

Reply:
Hmmmm, it seems clear to me already because I'm thinking that you need to add a slash if there isn't one on either the left of the right.